### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by vincerubinetti**
- add overview, links to webapp and reports, and "submit project" instructions to readme
- move pdf report list import to global file
- add page/route for listing all pre-generated pdf reports
- add notes and descriptions to app where appropriate
- new util css classes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
+# CFDE ICC Evaluation Core
+
+## Overview
+
+The [Common Fund Data Ecosystem (CFDE)](https://commonfund.nih.gov/dataecosystem) is an effort to bring together knowledge across [Common Fund](https://commonfund.nih.gov/) programs into a cohesive resource.
+The Integration and Coordination Center (ICC) is a center within the CFDE responsible for, among other things, evaluating the activities of Common Fund programs.
```
Comment (cgreene) [inline README.md:6]:
```suggestion
The Integration and Coordination Center (ICC) is a center within the CFDE responsible for, among other things, reporting on the impact and influence of the CFDE.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+
+This repository is a place to:
+
+- View a high-level evaluation of Common Fund program activity (e.g. projects over time, dollars awarded, # of publications produced, etc.).
```
Comment (cgreene) [inline README.md:10]:
```suggestion
- View a high-level dashboard of Common Fund and CFDE activities (e.g. projects over time, dollars awarded, # of publications produced, etc.).
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+- Submit details about your Common Fund project to be included in the above.
```
Comment (cgreene) [inline README.md:11]:
```suggestion
- Ensure your project's activities and materials are included in the above by aligning with the ingestion processes for the dashboard.
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,50 @@
...
+- Maintain code used to coordinate the above.
+
+The evaluation can be viewed as a live webapp dashboard or as separate PDF reports.
```
Comment (cgreene) [inline README.md:14]:
```suggestion
Reports are availabe as a live webapp dashboard or as PDFs that will be able to include narrative text and additional analyses.
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,50 @@
...
+
+[🖱️ The Webapp](cfde-eval.netlify.app)
+
+[📜 PDF Reports](cfde-eval.netlify.app/reports)
+
+## Submit your project
+
+We are able to get most details about Common Fund projects automatically from NIH resources, but there are some pieces of info that you must submit manually so that we have everything we need to create the evaluation.
```
Comment (cgreene) [inline README.md:22]:
```suggestion
We gather details about Common Fund projects automatically from NIH resources, but there are some pieces of info that require manual intervention to be integrated.
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,50 @@
...
+
+If you have a project under the Common Fund, please follow the instructions in the sections below as applicable.
```
Comment (cgreene) [inline README.md:24]:
```suggestion
If you would like your project's materials to be included, please follow the instructions in the sections below as applicable.
```

**Review by cgreene — APPROVED** [review]
a few suggestions - please let @seandavi take a look before merging as well to ensure he's ok with the content I modified.

File: README.md (hunk 4 above)
Comment (vincerubinetti) [reply README.md:14]: Throughout the docs and code comments, I'm trying to define "reports" as the printed pdfs and "pages" as the live dashboard pages. If you'd like me to use "reports" to mean either "pdf reports" or "webapp/page reports", that's okay but I'll want to change all instances of this throughout the repo for consistency (and avoid my own confusion).

File: README.md (hunk 3 above)
Comment (vincerubinetti) [reply README.md]: I committed something similar to this.

File: README.md (hunk 5 above)
Comment (vincerubinetti) [reply README.md]: I've commited something similar to this.

File: README.md (hunk 6 above)
Comment (vincerubinetti) [reply README.md]: I've commited something similar to this.

File: README.md (hunk 4 above)
Comment (cgreene) [reply README.md]: A change of that scale was not intended. :)

Comment (vincerubinetti) [reply README.md]: It won't take that long if you think trying to make those terms stick will be an uphill battle. Just want to be consistent either way.

**Pull request merged**